### PR TITLE
chore: Delete appengine-java8/multitenancy/

### PR DESCRIPTION
**Pull request by NimJay**
## Description

See Google-internal link: [b/348423641](http://b/348423641)

## Checklist

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [x] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by Sita04** [comment]
@NimJay Are the doc inclusions handled?

**Comment by NimJay** [comment]
@Sita04, thank you for the quick review! 

> Are the doc inclusions handled?

Good thought. I just left [a new comment in the Buganizer](b/348423641#comment2) to answer your question.
tl;dr - The docs already freeze the region tags at a specific commit.

Related question: Do I have to check for doc inclusion manually? In other words, would [snippet-bot](https://github.com/apps/snippet-bot) catch any doc inclusions for me.

**Review by iennae — APPROVED** [review]
